ui/lib/pki/addon/resolver.js

```diff
@@ -0,0 +1,3 @@
+import Resolver from 'ember-resolver';
```

Comment:
Can we delete this file if wherever we have `import Resolver from './resolver';` we change to `import Resolver from 'ember-resolver';`